### PR TITLE
Curve mapper sv_copy

### PR DESCRIPTION
**Pull request by zeffii**
addresses #3721

### CODE REVIEW

File: node_tree.py [hunk 1]
```diff
@@ -346,6 +353,7 @@ def copy(self, original):
             self.use_custom_color, self.color = settings
         self.sv_copy(original)
         self.n_id = ""
+        self.sv_copy_post(original)
```
Comment (Durman) [inline node_tree.py:356]:
```suggestion
        self.n_id = ""
        self.sv_copy(original)
```

Comment (zeffii) [reply node_tree.py:356]: if we never need to know the current node_id before copying, then yes, perhaps `sv_copy_post` is not needed.

Comment (zeffii) [reply node_tree.py]: ok , lets do that.

**Review by zeffii — COMMENTED** [review]
agreed

**Pull request merged**